### PR TITLE
[6] Develop a Product Page using HTML

### PR DESCRIPTION
**Pull request by john84edeh**
## Task: `[6] Develop a Product Page using HTML`
- `Updated the e-commerce site using the correct title`

### CODE REVIEW

File: submissions/edehJohn/html/index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<!Doctype html>
```
Comment (ToluAkin) [inline submissions/edehJohn/html/index.html:1]: Well done!
Why are you deleting your .txt file?
You did not add the images used in this file to this pull request?
I noticed you used more than 2 product images. You need to update your file @john84edeh

Comment (john84edeh) [reply submissions/edehJohn/html/index.html:1]: thank you ma. I dont remember deleting any txt file, and sorry, what i my suppose to update  on the file ?,. to down size the product images or add the images used in the PR? or both?

Comment (ToluAkin) [reply submissions/edehJohn/html/index.html:1]: You are using images on your html file but the images are not added to the pull request.

Comment (ToluAkin) [reply submissions/edehJohn/html/index.html:1]: Make sure you are using ONLY 2 product images. @john84edeh

**Pull request closed without merging**